### PR TITLE
update section titles & lines, linked "contact us"

### PR DESCRIPTION
**Pull request by vfatemi**
more descriptive section titles, added lines under ## sections, added link to "contact us" page for questions.

### CODE REVIEW

File: organizerguide.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 So, you are thinking of organizing a short virtual conference session. This is easy, rewarding, and takes little work. This manual will guide you through the process.
 
+## Main points to consider
+---
```
Comment (akhmerov) [inline organizerguide.md]: The line separators are meant to separate sections of a document. Are you sure one is appropriate here?

Comment (vfatemi) [reply organizerguide.md]: I felt that the headers of the different section levels were not distinguishable enough, so the lines are a suggestion to help make those demarcations more clear.

Comment (akhmerov) [reply organizerguide.md]: We should rather change the style his css at a later stage then

**Comment by akhmerov** [comment]
Also this is obsolete.

**Pull request closed without merging**